module_catalogue/infrastructure_substrate/eks.tf

```diff
@@ -6,6 +6,12 @@ data "aws_eks_cluster_auth" "eks" {
   name = module.eks.cluster_id
 }
 
+provider "kubernetes" {
```

Comment:
Please don't add this in here, we can add it as the `generate` step in the terragrunt.hcl if necessary

Reply:
Ok, I remove this part. as you wrote, miss the EKS cli!